sample_chorals/sumChildren.js

```diff
@@ -0,0 +1,16 @@
+function(children, done) {
+  /* Sample choral func.
+   * Sum every value of every child and return that
+   * value as the new value of this choral
+   */
+  var children = Object.values(children)
+  var sum = 0;
+  for(var i = 0; i < children.length; i++) {
+    var data = children[i].data;
+    var dataValues = Object.values(data);
+    for (var j = 0; j < dataValues.length; j++) {
```

Comment:
So this sums all the values of each member in the children.

Child1
{
   foo: 1,
   bar: 2
}

Child2
{
   foobar: 3,
   barfoo: 4
}

would output 

{
   sum_of_children: 10
}